query-connector/e2e/user_management.spec.ts

```diff
@@ -7,17 +7,26 @@ test.describe("User management", () => {
     await page.goto(TEST_URL);
   });
 
-  test("Users tab loads list of users", async ({ page }) => {
+  test("User management tab does not show for Standard users", async ({
+    page,
+  }) => {
     await page.getByRole("link", { name: "Try it out" }).click();
     await page.getByTestId("menu-button").click();
-    await page.getByRole("link", { name: "User Management" }).click();
-    await expect(
-      page.getByRole("heading", { name: "User management" }),
-    ).toBeVisible();
-    // Users tab is selected by default
-    await expect(page.getByRole("link", { name: "Users" })).toHaveClass(
-      /border-bottom-05 border-primary/,
-    );
-    await expect(page.getByText("No users found")).toBeVisible();
+    expect(
```

Comment:
Can this e2e check be added back? in theory the app should have auth disabled for playwright 🤔  so checking that the user management page loads is possible.